src/bufr/BufrParser/Exports/Export.cpp

```diff
@@ -0,0 +1,210 @@
+/*
+ * (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#include "Export.h"
+
+#include <ostream>
+#include <iostream>
+
+#include "eckit/exception/Exceptions.h"
+
+#include "Filters/GreaterFilter.h"
+#include "Filters/LesserFilter.h"
+#include "Filters/RangeFilter.h"
+#include "Splits/CategorySplit.h"
+#include "Variables/MnemonicVariable.h"
+#include "Variables/DatetimeVariable.h"
+#include "Variables/Transforms/Transform.h"
+#include "Variables/Transforms/TransformBuilder.h"
+
+
+namespace
+{
+    namespace ConfKeys
```

Comment:
Cool, I'll look into it and add Zenhub ticket. It would be really good to have YAML validation as you described. Currently I'm effectively validating the YAML in many different places manually... not the greatest design.

Reply:
Resolved by writing Zenhub issue.